docker/Makefile

```diff
@@ -36,15 +38,16 @@ build: prep
 		--build-arg version=$(version) \
 		--build-arg pupperware_analytics_stream=$(PUPPERWARE_ANALYTICS_STREAM) \
 		--file puppetdb/$(dockerfile) \
-		--tag puppet/puppetdb:$(version) \
+		--tag $(NAMESPACE)/puppetdb:$(version) \
 		..
 ifeq ($(IS_LATEST),true)
-	@docker tag puppet/puppetdb:$(version) puppet/puppetdb:latest
+	@docker tag $(NAMESPACE)/puppetdb:$(version) $(NAMESPACE)/puppetdb:latest
 endif
 
 test: prep
 	@bundle install --path .bundle/gems
-	@PUPPET_TEST_DOCKER_IMAGE=puppet/puppetdb:$(version) bundle exec rspec puppetdb/spec
+	@PUPPET_TEST_DOCKER_IMAGE=$(NAMESPACE)/puppetdb:$(version) \
+		bundle exec rspec puppetdb/spec
 
 publish: prep
 	@docker push puppet/puppetdb:$(version)
```

Comment:
I mentioned this in the commit description 😈 

No, because then we'd be pushing up personal/custom image like `nwolfe/puppetdb`.

Also you never would use this namespace override in CI, since we'd always want that to be `puppet`, and we shouldn't be pushing to DockerHub from our laptops so we shouldn't need to build+push locally ever.

Reply:
Yeah, I should have explained more. I was referring to running a local registry - i.e. https://docs.docker.com/registry/deploying/ - where you'd want to push / test from. We floated the idea of hosting local registries for dev workflows, but it's probably not too important right now. I could go either way in the end, but seemed like consistency in the script would be good, even if we never expect it to veer away from `puppet` when publishing.

If we need to host local publish / compose style workflows, we can always address that later.